go.mod

```diff
@@ -3,7 +3,7 @@ module terraform-provider-delphix
 go 1.17
 
 require (
-	github.com/delphix/dct-sdk-go v1.2.0
+	github.com/delphix/dct-sdk-go v1.6.0
```

Comment:
This is a big change from 1.2 to 1.6. Please document tests across resources to make sure this is not breaking anything

Reply:
I didnt quite get it. 
Few points:
1. when i upgraded to the new version, some of my code changes were suggested due to new functions added etc etc so all those changes are taken care of.
2. this PR contains just 1 feature so I was think post all the dev work, we can run the integration test suite that we have written in terraform and make apt changes if any breaking errors we encounter.

Reply:
For clarification: I was suggesting that in this PR you are bumping up the DCT version from 1.2 to 1.6. So it is safe to assume that many functions' behavior changes may or may not be caught by IDE. Hence I asked about sanity-checking all resources to make sure it is not breaking anything.  

If you are saying to run the integration test suite after this to achieve this, let's make sure 
1) the integration test covers all resources and popular resource blocks 
2) that such post-merge testing is documented in this PR for future ref. If it is easy to manually run that test suite before the merge, that would be preferred